src/bionics.cpp

```diff
@@ -286,6 +291,7 @@ void bionic_data::load( const JsonObject &jsobj, const std::string & )
     optional( jsobj, was_loaded, "exothermic_power_gen", exothermic_power_gen );
     optional( jsobj, was_loaded, "power_gen_emission", power_gen_emission );
     optional( jsobj, was_loaded, "coverage_power_gen_penalty", coverage_power_gen_penalty );
+    optional( jsobj, was_loaded, "block_unins", block_unins );
```

Comment:
```suggestion
    optional( jsobj, was_loaded, "cant_uninstal", cant_uninstal );
```
Maybe give it a more explicit name?